Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -117,7 +90,7 @@ public class NetworkClient : MonoBehaviour
         /// <summary>
         /// The host server
         /// </summary>
-        NetworkServer hostServer;
+        public NetworkServer hostServer;
```

Comment:
Why? Can't the component just set it by itself?

Reply:
ClientObjectManager.cs:539
`identity.Server = client.hostServer;`